layer/empty.go

```diff
@@ -24,6 +26,13 @@ func (el *emptyLayer) TarStream() (io.ReadCloser, error) {
 	return ioutil.NopCloser(buf), nil
 }
 
+func (el *emptyLayer) TarSeekStream() (ioutils.ReadSeekCloser, error) {
+	buf := new(bytes.Buffer)
+	tarWriter := tar.NewWriter(buf)
+	tarWriter.Close()
```

Comment:
How come we get a new `TarWriter` and then immediately close it? I understand that we don't want to actually write the tar, but why do we have to open and then close a `TarWriter`?